ansi/renderer_test.go

```diff
@@ -56,6 +57,7 @@ func TestRenderer(t *testing.T) {
 			goldmark.WithExtensions(
 				extension.GFM,
 				extension.DefinitionList,
+				emoji.Emoji,
```

Comment:
Don't we also need to add this extension when initializing `goldmark` in `glamour.go`'s `NewTermRenderer` function? I wonder if this should be an optional setting, like `WithWordWrap`.

Reply:
I believe I did add the extension to `NewTermRenderer` at line 66 of `glamour.go`, maybe I missed a different location? 

I think this would be easy enough to make into an option setting named `WithEmoji`. In my use case I always want it enabled which is why I went the current direction. Happy to make the change, just let me know.

Reply:
No, you did not :smiley:  I could swear this change wasn't showing up in the GitHub UI when I reviewed the PR earlier today... but I guess I must have overlooked it, even though I started searching for it.

Sorry for the confusion!

Reply:
Yeah, I think `WithEmoji` sounds good!

Reply:
I updated emoji rendering to be optional using the `WithEmoji` TermRendererOption and added a test for it. Let me know if you would like any other changes made 🙇